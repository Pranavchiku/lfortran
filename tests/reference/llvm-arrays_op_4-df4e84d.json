{
    "basename": "llvm-arrays_op_4-df4e84d",
    "cmd": "lfortran --no-color --show-llvm {infile} -o {outfile}",
    "infile": "tests/../integration_tests/arrays_op_4.f90",
    "infile_hash": "217baeb5f0a4438c9af1044fbcb9991796f27fdc7f8cb69be5213c8f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "llvm-arrays_op_4-df4e84d.stdout",
    "stdout_hash": "29c1854725d0ddee6ee80dd3afdfb8a8656f55147e70627f432d741c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}