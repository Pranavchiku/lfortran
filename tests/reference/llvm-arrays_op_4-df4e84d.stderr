warning: Implied save feature
 --> tests/../integration_tests/arrays_op_4.f90:6:1
  |
6 | integer :: i, j, k, dim1 = 10, dim2 = 100, dim3 = 1
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/arrays_op_4.f90:6:1
  |
6 | integer :: i, j, k, dim1 = 10, dim2 = 100, dim3 = 1
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/arrays_op_4.f90:6:1
  |
6 | integer :: i, j, k, dim1 = 10, dim2 = 100, dim3 = 1
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
